docs/source/ros_api.rst

```diff
@@ -0,0 +1,117 @@
+.. Copyright 2016 The Cartographer Authors
+
+.. Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+..      http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+=======
+ROS API
+=======
+
+The following ROS API is provided by `cartographer_node`_.
+
+Command-line Flags
+==================
+
+.. TODO(damonkoler): Use an options list if it can be made to render nicely.
+
+\-\-configuration_directory
+  First directory in which configuration files are searched, "second is always
+  the Cartographer installation to allow including files from there.
+
+\-\-configuration_basename
+  Basename, i.e. not containing any directory prefix, of the configuration file.
```

Comment:
", e.g. backpack_3d.lua"

makes it clear that this is were the lua files are mentioned.